### PR TITLE
Updated component version

### PR DESCRIPTION
**Pull request by dislbenn**
Signed-off-by: dislbenn <dbennett@redhat.com>

**Related Issue:**  stolostron/backlog#<ISSUE_NUMBER>

### CODE REVIEW

**Review by jlpadilla — APPROVED** [review]
/lgtm

**Comment by dislbenn** [comment]
Adding a hold so we can fix the `ci/prow/publish` job: https://github.com/openshift/release/pull/31875.

File: Makefile
Comment (jlpadilla) [inline Makefile]: I think we can remove the `-v2` to match the quay repo name.

File: Makefile
Comment (jlpadilla) [inline Makefile]: This works for this PR, but let's talk about consolidating this with `COMPONENT_VERSION`, I believe that it's not required to match the ACM release.

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold

File: Makefile
Comment (dislbenn) [reply Makefile]: Removed `-v2`.

**Comment by dislbenn** [comment]
@jlpadilla this PR should be ready to merge. Please let me know if there are any more changes needed.

File: Makefile
Comment (jlpadilla) [inline Makefile:8]: Here are some thoughts about the VERSION. We may need to make some changes, but I don't want to hold this PR.
- For Github commit sha is preferred over date because we can correlate the the bundle to a specific commit.
- This version will also need to match the version in the CSV [here](https://github.com/stolostron/search-v2-operator-bundle/blob/main/bundle/manifests/search-v2-operator.clusterserviceversion.yaml#L317).  I'll need to check if the CSV accepts the postfix.

**Review by jlpadilla — APPROVED** [review]
/lgtm

We'll iterate over the VERSION in the next PR.

**Comment by dislbenn** [comment]
Merging this PR to unblock other PRs.

File: Makefile
Comment (dislbenn) [reply Makefile:8]: Adding this change within a separate PR.

**Pull request merged**
